tsl/test/expected/cagg_plugin.out

```diff
@@ -0,0 +1,308 @@
+--
+(1 row)
+
+-- Generate a few entries in one go. This caused some problems
+-- initially, so check this first.
+INSERT INTO conditions VALUES ('2020-05-01 12:34:56Z', 98, 37.0);
+INSERT INTO conditions VALUES
+       ('2020-05-02 12:34:56Z', 98, 37.0),
+       ('2020-05-02 13:45:00Z', 98, 37.0);
+SELECT * FROM get_invalidations('my_slot', 'conditions', 'recorded_at');
+ hypertable_id | lowest_modified_value  | greatest_modified_value 
+---------------+------------------------+-------------------------
+             1 | 2020-05-01 12:34:56+00 | 2020-05-01 12:34:56+00
+             1 | 2020-05-02 12:34:56+00 | 2020-05-02 13:45:00+00
+(2 rows)
+
+-- Generate a single invalidation line and check how that appears.
+INSERT INTO conditions VALUES ('2020-05-01 12:34:56Z', 99, 37.0);
+SELECT * FROM get_invalidations('my_slot', 'conditions', 'recorded_at');
+ hypertable_id | lowest_modified_value  | greatest_modified_value 
+---------------+------------------------+-------------------------
+             1 | 2020-05-01 12:34:56+00 | 2020-05-01 12:34:56+00
+(1 row)
+
+INSERT INTO conditions VALUES
+       ('2020-05-02 12:34:56Z', 99, 37.0),
+       ('2020-05-02 13:45:00Z', 99, 37.0);
+-- Check that we got the invalidations
+SELECT * FROM get_invalidations('my_slot', 'conditions', 'recorded_at');
+ hypertable_id | lowest_modified_value  | greatest_modified_value 
+---------------+------------------------+-------------------------
+             1 | 2020-05-02 12:34:56+00 | 2020-05-02 13:45:00+00
+(1 row)
+
+-- This should generate an invalidation including the full range since
+-- it is inside a transaction.
+INSERT INTO conditions
+SELECT recorded_at, 99, random()*80 - 40
+  FROM generate_series('2025-02-01'::timestamptz,
+  	               '2025-03-31'::timestamptz,
+		       '1 minute'::interval) AS recorded_at;
+SELECT * FROM get_invalidations('my_slot', 'conditions', 'recorded_at');
+ hypertable_id | lowest_modified_value  | greatest_modified_value 
+---------------+------------------------+-------------------------
+             1 | 2025-02-01 00:00:00+00 | 2025-03-31 00:00:00+00
+(1 row)
+
+UPDATE conditions SET temp = 0.0 WHERE device_id = 99;
+SELECT * FROM get_invalidations('my_slot', 'conditions', 'recorded_at');
+ hypertable_id | lowest_modified_value  | greatest_modified_value 
+---------------+------------------------+-------------------------
+             1 | 2020-05-01 12:34:56+00 | 2025-03-31 00:00:00+00
+(1 row)
+
+DELETE FROM conditions WHERE device_id = 99;
+SELECT * FROM get_invalidations('my_slot', 'conditions', 'recorded_at');
+ hypertable_id | lowest_modified_value  | greatest_modified_value 
+---------------+------------------------+-------------------------
+             1 | 2020-05-01 12:34:56+00 | 2025-03-31 00:00:00+00
+(1 row)
+
+-- Test that we can do multiple changes (in different transactions) in
+-- a suite and get the changes for that using the plugin.
```

Comment:
These transactions are all serially executed in the same backend. I think we should also have an isolation test that uses different backends to modify data and then read invalidations where some modifying transactions are committed and some are not. 

I guess the plugin should see invalidations from committed backends but not ongoing backends in that case?